charts/skypilot/templates/rbac.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: skypilot-api-sa
+  namespace: {{ .Release.namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: skypilot-api-role
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
```

Comment:
Shall we expose policy to helm values? Keeping it tunable until we figure out the minimum privileges will be helpful for environments with some security restrictions.

Reply:
That's a good idea, thanks! Added now.